samples/apps/logging/logging.cpp

```diff
@@ -12,6 +12,10 @@
 #include "ccf/http_query.h"
 #include "ccf/user_frontend.h"
 #include "ccf/version.h"
+// FIXME: The header below is used for a single check of the certificate and
+// could be done using OpenSSL. For now, we keep it as is, but we should make
+// the change once we deprecate, and remove, mbedTLS.
+#include "crypto/mbedtls/mbedtls_wrappers.h"
```

Comment:
I would suggest grouping in a ticket all the stuff that needs cleaning up when mbedTLS goes, especially what won't easily be grepped.

Reply:
They're all in the README file. I have removed all the FIXME's for now and left the comments.

This specific comment will be noted when I try to remove the header, so not going to be forgotten.